machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts

```diff
@@ -62,7 +62,6 @@ const model = createModel(
     transactionId: '',
     bindingTransactionId: '',
     revoked: false,
-    downloadCounter: 0,
     maxDownloadCount: 10,
```

Comment:
Maximum download count should not be hardcoded, it should be fetched from inji-default.properties.
we can use getAllConfiguration method to get the property and use it here.

Reply:
maxDownloadCount is assigning from the inji-default.properties